src/schematools/types.py

```diff
@@ -737,6 +738,29 @@ def get_fk_fields(self) -> Iterator[str]:
             if field.relation:
                 yield field.name
 
+    @property
+    def zoom(self) -> ZoomLevel:
+        """Returns (minzoom, maxzoom) levels at which non-geometric fields are included in MVT.
+
+        Minimum and maximum are understood numerically, so 0 (entire earth) is the lowest level.
+        The range includes both endpoints.
+
+        The zoom levels may be strings, in which case they reference object fields that in turn
+        contain the levels: if the return value is (0, "maxzoom"), then an object x is shown at
+        levels 0 through x.maxzoom, inclusive.
+        """
+        z = self.get("zoom")
+        if z is None:
+            return _default_zoomlevel
+
+        return ZoomLevel(
+            min=z.get("min", _default_zoomlevel.min), max=z.get("max", _default_zoomlevel.max)
+        )
+
+
+ZoomLevel = typing.NamedTuple("ZoomLevel", min=Union[int, str], max=Union[int, str])
```

Comment:
```suggestion
class ZoomLevel(typing.NamedTuple):
  min: Union[int, str]
  max: Union[int, str]
```

?

Reply:
Good question from @vdboor. The `zoom` function, being the sole creator of `ZoomLevel` instances, should be able to cast the attributes to `int`.

If there still is a good reason for `min` and `max` to be both `int` and `str` it should be documented as such with an appropriate docstring.

Reply:
Dat staat in de docstring voor `zoom`: dit kan een zoomniveau zijn of de naam van een veld waarin het zoomniveau per object wordt opgezocht. Ik stop liever niet te veel documentatie op dit ding; het is puur een manier om `.min` en `.max` te kunnen doen ipv. een tuple uit te pakken in potentieel de verkeerde volgorde.